packages/shared/styles/components/SfAddToCart.scss

```diff
@@ -13,7 +13,7 @@ $add-to-cart__select-quantity-max-width: 5.625rem !default;
   &__button {
     flex: 1;
     &:disabled {
-      background-color: $c-accent-secondary;
+      background-color: $c-primary-variant;
```

Comment:
@EduardDopler the second one is the false contrast ratio result :D, the third one is not an option though. In case `disabled`, the best is gray color rather than any green tmho

Reply:
I meant the first _disabled_ button (so the second button in the list) 🙈 very confusing statement, sorry...
anyway, I agree: disabled buttons can be simply grey–that solves the problem 👍